### DISCUSSION TITLE
Provide images for inscriptions

### DISCUSSION THREAD

**Discussion by SophiaD-M** [SChAth/ScAthiop#1853] [Ideas]
Would it be possible, or is it planned for the future, to provide also images to (RIE) inscriptions? 
I know they mightbe less available, or only old b/w copied from older publications, but it would be helpful for sure.

**Comment by PietroLiuzzo** [comment]
Thanks @SophiaD-M it is an interesting idea, we can certainly store them and serve them as the others, but I think you are right, the issue is here primarily availability.

**Reply by SophiaD-M** [reply]
Maybe for the old DAE material one could at least link to the images in the books on archive.org? (naive questions, I know)
Just a thought

**Reply by PietroLiuzzo** [reply]
yes, you can do that! This page of the [Guidelines](https://betamasaheft.eu/Guidelines/?q=images&id=images) should show how. archive.org supports IIIF so, the links can be provided to the exact pages.
